pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -320,15 +234,129 @@ stdenv.mkDerivation {
 
     + optionalString cc.langVhdl or false ''
       ln -s $ccPath/${prefix}ghdl $out/bin/${prefix}ghdl
+    '';
+
+  propagatedBuildInputs = extraPackages;
```

Comment:
This was tantamount `propagatedNativeBuildInputs` before, but it doesn't (currently) make a difference with native, and this is more correct with cross.